app/lib/pre_assembly/bundle.rb

```diff
@@ -579,7 +578,7 @@ def process_digital_objects
           dobj.pre_assemble
           # Indicate that we finished.
           dobj.pre_assem_finished = true
-          log_and_show "Completed #{dobj.druid.druid}"
+          log_and_show "Completed #{dobj.druid}"
```

Comment:
Note that `dobj.druid.druid` did not work on the manifest approach I used (for existing test_data folder contents) so I changed it to `dobj.druid`